metricbeat/module/docker/vendor/github.com/fsouza/go-dockerclient/container.go

```diff
@@ -99,6 +99,21 @@ func (p Port) Proto() string {
 	return parts[1]
 }
 
+// HealthCheck represents one check of health.
+type HealthCheck struct {
```

Comment:
if the docker library is updated, also the commit id in the glide.yml file must be updated.